ironfish-cli/src/utils/chainport/requests.ts

```diff
@@ -0,0 +1,64 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import axios from 'axios'
+import { getConfig } from './config'
+import {
+  ChainportBridgeTransaction,
+  ChainportNetwork,
+  ChainportTransactionStatus,
+  ChainportVerifiedToken,
+} from './types'
+
+export const fetchChainportTransactionStatus = async (networkId: number, hash: string) => {
+  const config = getConfig(networkId)
+  const url = `${config.endpoint}/api/port?base_tx_hash=${hash}&base_network_id=22`
+
+  const response: {
+    data: ChainportTransactionStatus
+  } = await axios(url)
```

Comment:
nit -- should we use this as `axios.get` for consistency?

Reply:
Done